src/SlideCaption.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="VueCarousel-caption">
+    <aside class="slide-caption" @click="handleClick">
```

Comment:
I think I used a `slide-*` class on accident somewhere, but I was trying to keep all the Vue component classes prefixed with `VueCarousel-*`, then was using the `slide-*` in the actual Elixir app.